### ISSUE TITLE
Problem using Tool menu in Thor

### ISSUE THREAD

**Issue by CHamlinPNP** [VFPX/Thor#34]
## 📝 Provide detailed reproduction steps (if any)

1. … Thor Tools -> Favorites, MRUs, etc -> Open Projects
2. … Get an error in ToolCode procedure in thor_tool_peme_openproject.prg
3. … Unrecognized command verb on line "_oPEMEditor.oUtils."

### ✔️ Expected result

Since oUtils is an object pointing to the various utilities...I would assume that the name of an utility would follow the last period of the statement. I just don't know what utility it is trying to run...and I would think that someone would want to correct the code for the next person.

### ❌ Actual result

Receive an error 16

## 📷 Screenshots

_Are there any useful screenshots? WinKey+Shift+S and then just paste them directly into the form_

![image](https://user-images.githubusercontent.com/34104227/168319376-c8623472-84e5-4c5e-9cbd-5dd935537b6f.png)

**Comment by rschummer** [comment]
I'm not sure where this code originated. I have the latest Thor Repository and I don't have the Open Projects under the MRU menu. If you Edit Tool via the Thor Configuration, are you prompted to make a copy in your My Tools folder (or optionally open it read only) or does it open up directly in the code editor? 

Maybe this was something that was not finished and accidentally released in an older version and was pulled from the Thor Repository before I grabbed an update.

Rick

**Comment by rschummer** [comment]
I just did a search on the PEM Editor source code. You should be able to get this to work with the following line of code:

_oPEMEditor.oUtils.DoFormModifyFile('Project')

However, all it does is MODIFY PROJECT ? NOWAIT, it does not present a MRU list like you would see with MODIFY PROJECT in the Command Window.

**Comment by CHamlinPNP** [comment]
I have Thor to do an update daily so it should not be out of date. I will try to reinstall Thor and see if I get anything different. I will report back with my findings.

**Comment by CHamlinPNP** [comment]
I installed a totally clean copy of Thor in a different location. I had many issues including...it could not run the install_zproc.prg and it could not run the Code References install.prg.

Having said that...it also shows that 4 other issues during compile of PRGs.

![image](https://user-images.githubusercontent.com/34104227/168426887-f8ed8170-8976-43cd-8993-133752326232.png)
![image](https://user-images.githubusercontent.com/34104227/168426923-8176def3-31f4-4b42-954a-b045f9d43ccb.png)
![image](https://user-images.githubusercontent.com/34104227/168426942-ea18d099-78c1-4257-a798-82b90af69c84.png)
![image](https://user-images.githubusercontent.com/34104227/168426960-4b5d1d21-7c1f-48e7-8daf-5a5d82984434.png)

The error for OpenProject that I originally wrote about...the err file is there so it did exist...but I cannot find the PRG or FXP any longer. My guess is the .Err must be in some zip which had the error at one point. I am also guessing that Thor does not remove any programs....so at one point I had the thor_tool_peme_openproject.prg which was previously there...Thor did not touch it...so it remained there with the issue. Something needs to be thought about how to remove features as things are developed.

**Comment by DougHennig** [comment]
I looked into some of these issues:

- Thor_Tool_Repository_VCD4PRG.h is indeed missing and I have no idea where to find it. However, unless you use that tool (I'm not even sure what it is), that won't cause a problem.
- I fixed the typo in Thor_Tool_Finder.prg
- The typo in Thor_Tool_GoFish5.prg is an odd one: somehow, an extra ENDPROC is being inserted into the code and I'm not sure where. However, it doesn't prevent it from running properly so no harm done.
- I haven't been able to find Thor_Tool_PEME_OpenProject.prg. The problem is that the source for Thor is spread out over several projects: Thor, Thor Repository, and PEM Editor, plus others I think. That being said, Open Project doesn't even appear in my Thor menu.
